projects/plugins/data_providers/mongodb-datasource-provider/src/main/java/edu/emory/cci/bindaas/datasource/provider/mongodb/MongoDBQueryHandler.java

```diff
@@ -40,7 +40,7 @@
     private Log log = LogFactory.getLog(getClass());
     private OutputFormatRegistry registry;
     private JsonParser parser = new JsonParser();
-    
+    public final static String ROLE = "role";
```

Comment:
This constant "role" also should go somewhere higher as it will be reused by multiple providers.

Reply:
I will use the same constant which I will be using in other files.